google_takeout_parser/models.py

```diff
@@ -105,13 +105,27 @@ def key(self) -> int:
 class Location(BaseEvent):
     lng: float
     lat: float
+    address: Optional[str]
     accuracy: Optional[int]
     dt: datetime
+    placeId: Optional[str]
 
     @property
     def key(self) -> Tuple[float, float, Optional[int], int]:
         return (self.lng, self.lat, self.accuracy, int(self.dt.timestamp()))
 
+@dataclass
+class PlaceVisit(BaseEvent):
+    location: Location
+    lng: float
+    lat: float
+    startTime: datetime
+    endTime: datetime
+    visitConfidence: Optional[int]
+
+    @property
```

Comment:
Should probably include `startTime` in the `key`. Very unlikely you get the same lat/lon/visitconfidence pair twice, but would be nice to uniquely identify things by datetime when trying to debug merges instead of trying to search for a visitConfidence/lat/lon.